portfolio/src/main/webapp/index.html

```diff
@@ -9,13 +9,27 @@
   <body>
     <div id="content">
       <h1>Ryan's Portfolio</h1>
-      <p>Hello! My name is Ryan Simone and I am currently a Google Step Intern. I am a rising junior at the University of California Santa Barbara pursuing a Bachelor's degree in Computer Science.</p>
-      <p>I was born in Torrance, CA on August 18, 1999. I am the youngest, yet tallest, member of my family. As a child, I was nicknamed "Ry-Ry the beach going guy" because of my love for the beach. I played competetive soccer until college. </p>
-      <p>To visit my github page, click 
+      <p>Hello! My name is Ryan Simone and I am currently a Google Step Intern.</p>
+      <p>I am a rising junior at the University of California Santa Barbara pursuing a Bachelor's degree in Computer Science.</p>
+      <p>I'm the youngest, yet tallest, member of my family. I've lived in Southern California my entire life.</p>
+      <h2>Hobbies</h2>
+      <dl>
+          <dt>I love watching sports. My favorite is soccer.</dt>
+          <dt>I enjoy skateboarding and surfing.</dt>
+          <dt>I listen to music constantly. My favorite genre is surf rock.</dt>
+      </dl>
+      <h2>Projects</h2>
+      <dl>
+          <dt>Create a shared music queue web app with some classmates.</dt>
+          <dt>Created a smart calendar web app with a group of friends to help improve peoples' busy schedules.</dt>
+          <dt>Created an LA sports fan website with my brother.</dt>
+      </dl>
       <img src="images/IMG-3136.jpg" id="img"/>
       <p>Click here to get a random fact about me:</p>
       <button onclick="addRandomFact()">Get Fact</button>
       <div id="fact-container"></div>
+      <p> You can visit my github page <a href="https://github.com/rysimone">here</a>.</p>
```

Comment:
If the user is using a screen reader and cycling through the set of links on the page, this one will just be read as "here", which lacks context. You can make this more accessible by setting the `aria-label` attribute to be something like "Link to my github page", or by rephrasing the item itself, e.g. `You can visit <a>my github page</a>.`.

More info: https://www.washington.edu/accessibility/links/

Reply:
I never thought of that. Glad we are having this accessibility talk today.